naff/models/naff/extension.py

```diff
@@ -86,6 +86,8 @@ def __new__(cls, bot: "Client", *args, **kwargs) -> "Extension":
                         bot.add_modal_callback(val)
                     elif isinstance(val, naff.ComponentCommand):
                         bot.add_component_callback(val)
+                    elif isinstance(val, naff.HybridCommand):
+                        bot.add_hybrid_command(val)
```

Comment:
You forgot to handle commands being *un*loaded from extensions

Reply:
Fixed! Though the fix is pretty complex 😅.